rheannemcintosh/london-cocktail-week-2024
# LCW24-5 | Save the bars into a CSV

# [LCW24-5] | Save the bars into a CSV

## Work
This work saves all of the bars to a CSV.

## Testing
- Run `jupyter-lab`
- Run all cells.
- Ensure the bar has saved to a CSV.

[LCW24-5]: https://rheannemcintosh.atlassian.net/browse/LCW24-5?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
